spec/compiler/crystal/tools/context_spec.cr

```diff
@@ -1,26 +1,24 @@
 require "spec"
 require "yaml"
-require "../../../../src/compiler/crystal/**"
```

Comment:
using relative paths allow testing the code of the compiler without building the compiler.
if we keep the require "compiler/crystal/**" the code that will be tested is the one in the compiler used to build the specs.

I would leave the require "../../.. ...."  since it allows more unit test like of the compiler.

Checking in the code, I notice that it could be replaced with `require "../../spec_helper"` But maybe that could be handle in another PR and just leave this for enforcing the `private` methods/classes.